source/sessions/driver-sessions.rst

```diff
@@ -859,8 +871,11 @@ Algorithm to return a ServerSession instance to the server session pool
    from the end of the queue and discarded (or allowed to be garbage collected)
 
 2. Then examine the server session that is being returned to the pool and:
+    * If this session is marked dirty (ie it was involved in a network error)
```

Comment:
"i.e.", unless "ie" is commonly used elsewhere in specs.

Reply:
Done